apps/site/assets/ts/schedule/ScheduleStore.tsx

```diff
@@ -0,0 +1,57 @@
+import { createStore } from "redux";
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+const storeReducer = (
+  state: any,
```

Comment:
@alicia-mbta we should be able to write a more specific type than `any` in this file. It's best to avoid using `any` whenever possible because you lose the benefit of having a static type system when you use it.